integrationtests/kubernetes_test.go

```diff
@@ -32,12 +28,14 @@ func TestKubernetesDeployment(t *testing.T) {
 
 	// retryable errors in terraform testing.
 	// generate ssh keys for test
-	tests.SSHKeys()
-	publicKey := os.Getenv("PUBLICKEY")
+	pk, sk, err := tests.GenerateSSHKeyPair()
+	if err != nil {
+		t.Log(err)
```

Comment:
at this point we should do `t.Fatal` because we can't possibly continue the test without generating the keys, also use `publicKey, privateKey` as names instead of pk and sk. Please make sure when you do setup and teardown to t.Fatal when you can't generate the ssh keys.